apps/ejabberd/include/ejabberd.hrl

```diff
@@ -18,8 +18,6 @@
 %%% 02111-1307 USA
 %%%
 %%%----------------------------------------------------------------------
--ifndef(MONGOOSEIM_EJABBERD_HRL).
```

Comment:
Wasn't such a line useful? We include ejaberd.hrl from almost every file. Maybe I don't understand the prev mechanism correctly, but to mean it meant - do not define the macros if they are already defined.